docs/dns_disruption.md

```diff
@@ -18,7 +18,7 @@ Without the OnInit parameter, we target all port 53 udp traffic **of each contai
 
 ## Forwarding non-matched requests
 
-Depending on your DNS setup you might need to override the DNS server and/or instruct the controller to forward requests to kube-dns. See the [advanced installation guide](installation.md#dns-resolution).
+Depending on your DNS setup you might need to override the DNS server and/or instruct the controller to forward requests to kube-dns. See the [configuration guide](configuration.md).
```

Comment:
Good catch on this now missing link. I think it might be preferable to link directly to the values.yaml entries you modified, rather than the generic configuration.md? This is a very soft preference though

Reply:
IMO: It make sense 👍  What do you think @Devatoria?